readthedocs/templates/projects/domain_list.html

```diff
@@ -12,26 +12,35 @@
 {% block project_edit_content_header %}{% trans "Domains" %}{% endblock %}
 
 {% block project_edit_content %}
-  {# Remove anytime after 2018-10-01 #}
-  <p class="empty">{% blocktrans %}Recently we added HTTPS support for custom domains. Depending on your setup, you may need to take action for your domain to support HTTPS. <a href="https://docs.readthedocs.io/en/latest/alternate_domains.html#custom-domain-ssl">See our docs</a> for details.{% endblocktrans %}</p>
-
   <p class="help_text">
     {% trans "This allows you to add domains to your project. This allows them to live in the same namespace in the URLConf for a subdomain or CNAME." %}
   </p>
 
   {% if object_list %}
-  <h3> {% trans "Existing Domains" %} </h3>
-  <p>
-    <ul>
-    {% for domain in object_list %}
-        <li class="{% if domain.machine %}domain-machine{% endif %} {% if domain.canonical %}domain-canonical{% endif %}" >
-        {{ domain.domain }}
-        (<a href="{% url 'projects_domains_edit' project.slug domain.pk %}">{% trans "Edit" %}</a>)
-        (<a href="{% url 'projects_domains_delete' project.slug domain.pk %}">{% trans "Remove" %}</a>)
-        </li>
-    {% endfor %}
-    </ul>
-  </p>
+    <div class="module">
+      <div class="module-wrapper">
+        <div class="module-header">
+          <h3>{% trans "Existing Domains" %}</h3>
+        </div>
+        <div class="module-list">
+          <div class="module-list-wrapper">
+            <ul>
+              {% for domain in object_list %}
+                <li class="module-item">
+                <span class="module-item-title">
+                  {{ domain.domain }}
+                </span>
+                <ul class="module-item-menu">
+                  <li><a href="{% url 'projects_domains_edit' project.slug domain.pk %}">{% trans "Edit" %}</a></li>
+                  <li><a href="{% url 'projects_domains_delete' project.slug domain.pk %}">{% trans "Remove" %}</a></li>
```

Comment:
Can you upload a screenshot to see how this looks like?

Reply:
Yes.
Here it is

![screencapture-localhost-8000-dashboard-atom-domains-2019-01-14-23_00_36](https://user-images.githubusercontent.com/29149191/51129539-bb47b580-1850-11e9-91cb-711f5a71469f.png)